service/internal/proctelemetry/config.go

```diff
@@ -171,7 +187,69 @@ func initPeriodicExporter(_ context.Context, exporter telemetry.MetricExporter)
 		if err != nil {
 			return nil, nil, err
 		}
-		return sdkmetric.NewPeriodicReader(exp), nil, nil
+		return sdkmetric.NewPeriodicReader(exp, opts...), nil, nil
+	}
+	if exporter.Otlp != nil {
+		var err error
+		var exp sdkmetric.Exporter
+		switch exporter.Otlp.Protocol {
+		case protocolProtobufHTTP:
+			exp, err = initOTLPHTTPExporter(ctx, exporter.Otlp)
+		case protocolProtobufGRPC:
+			exp, err = initOTLPgRPCExporter(ctx, exporter.Otlp)
+		default:
+			return nil, nil, fmt.Errorf("unsupported protocol %s", exporter.Otlp.Protocol)
+		}
+		if err != nil {
+			return nil, nil, err
+		}
+		return sdkmetric.NewPeriodicReader(exp, opts...), nil, nil
 	}
 	return nil, nil, fmt.Errorf("no valid exporter")
 }
+
+func initOTLPgRPCExporter(ctx context.Context, otlpConfig *telemetry.OtlpMetric) (sdkmetric.Exporter, error) {
+	opts := []otlpmetricgrpc.Option{}
+
+	if len(otlpConfig.Endpoint) > 0 {
+		opts = append(opts, otlpmetricgrpc.WithEndpoint(otlpConfig.Endpoint))
```

Comment:
should this be a required option?

Reply:
> By default, if an environment variable is not set, and this option is not passed, "localhost:4317" will be used. 

It is not required: https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc#WithEndpoint